src/components/campaigns/CreateCampaign.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { firebase, dbCampaigns } from '../../firebase';
+import { Button, Form , Modal, Segment } from 'semantic-ui-react'
+
+class CreateCampaign extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      campaigns: this.props.items
+    };
+
+    this.handleTitleChange = this.handleTitleChange.bind(this);
+    this.handleDescriptionChange = this.handleDescriptionChange.bind(this);
+    this.handleNeededAmountChange = this.handleNeededAmountChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleTitleChange(event) {
+    this.setState({title: event.target.value});
+  }
+
+  handleDescriptionChange(event) {
+    this.setState({description: event.target.value});
+  }
+
+  handleNeededAmountChange(event) {
+    this.setState({neededAmount: event.target.value});
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+    const campaign = {
+      id: this.props.items.length,
+      title: this.state.title,
+      description: this.state.description,
+      currentAmount: 10,
+      neededAmount: this.state.neededAmount,
+      image: 'http://via.placeholder.com/350x150'
+    }
+    dbCampaigns.doCreateCampaigns(campaign.id, campaign.title,campaign.description,campaign.currentAmount,campaign.neededAmount, campaign.image)
+  }
+
+  render() {
+    return (
+      <Modal trigger={<Button>Create Campaign</Button>} closeIcon>
+        <Modal.Header>
+          <h1>Create Campaign</h1>
+        </Modal.Header>
+        <Segment>
+          <Form onSubmit={this.handleSubmit}>
+            <Form.Group widths='equal'>
+              <Form.Input label="Title" type="text" value={this.state.value} onChange={this.handleTitleChange} />
```

Comment:
You can write:
onChange={event => this.setState({title: event.target.value})}

Then you do not need a function handleTitleChange(event) and this.handleTitleChange = this.handleTitleChange.bind(this);